src/opers.cc

```diff
@@ -654,7 +654,7 @@ static Obj LookupHashTable(Obj ht, Int hash, Obj flags)
 }
 
 template <Int len>
-static void StoreHashTable(Obj ht, Int hash, Obj new_flags, Obj new_with)
+static void StoreHashTable(Obj ht, Int hash, Obj new_with, Obj new_flags)
```

Comment:
Thanks. Perhaps should also add GAP_ASSERT calls to verify the types of those arguments?